docs/team/contributors.yaml

```diff
@@ -26,6 +26,12 @@
   team: active
   last-check-in: 2022-10
 
+- name: Eric Charles
+  handle: "@echarles"
+  affiliation: Datalayer
```

Comment:
cc @echarles to make sure your affiliation is correct here

Reply:
Thx @jtpio. Yes, affiliation is correct.